llvm/test/CodeGen/SPIRV/constant/global-constants.ll

```diff
@@ -1,5 +1,8 @@
 ; RUN: llc -O0 -mtriple=spirv32-unknown-unknown %s -o - | FileCheck %s
 
+target datalayout = "e-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024"
+target triple = "spir64"
```

Comment:
Does the triple still need to be added explicitly in the module (it is also passed as a flag to llc)? Also there is a mismatch between the triple in the flag and the triple declared here.

Reply:
I fixed tests by removing explicit targets and adding spirv-val where it was missed. Thank you for the suggestion.